.circleci/config.yml

```diff
@@ -98,13 +101,25 @@ jobs:
           name: Check PR Metadata
           command: bundle exec danger || echo "danger failed"
       - *cache_restore_rubocop
-      - run: bundle exec fastlane execute_tests
+      - run:
+          name: bundle exec fastlane execute_tests
+          command: |
+            RUBYOPT=<< parameters.ruby_opt >> bundle exec fastlane execute_tests
```

Comment:
For some reasons, I can't pass `RUBYOPT`  via CircleCI's environment variable settings😭 So the inline environment variable  was the only way I could do this.

Reply:
See this commit including changes to use environment variable settings in CircleCI and revert some fixes to see if warning detection works...
https://github.com/fastlane/fastlane/commit/fd7541ccff65ed3f852a5f094a0472bc91461461

Apparently, rspec with the above commit didn't emit any deprecation warnings so that this build just passed the check which was supposed to fail though.  
https://app.circleci.com/pipelines/github/fastlane/fastlane/2263/workflows/c94de87e-5700-4279-adb8-f94ddbef0f70/jobs/62797